lib/src/sass/_reset.scss

```diff
@@ -1,152 +1,343 @@
-@import 'variables';
-
-/* http://meyerweb.com/eric/tools/css/reset/ 
-   v2.0 | 20110126
-   License: none (public domain)
-*/
-*,
-*::before,
-*::after {
-    box-sizing: border-box;
-}
-
-div,
-span,
-applet,
-object,
-iframe,
-h1,
-h2,
-h3,
-h4,
-h5,
-h6,
-p,
-blockquote,
-pre,
-a,
-abbr,
-acronym,
-address,
-big,
-cite,
+/*! normalize.css v8.0.0 | MIT License | github.com/necolas/normalize.css */
```

Comment:
We made quite a few changes to fix issues that were introduced when we added the reset style. It looks like we are replacing the entire file. A few things with this big of change:
1. Why are we switching our reset/normalize file?
2. We will probably need a style review by @andrew-frueh to make sure this doesn't introduce any new bugs.

Reply:
The reason I replaced it was that I was experiencing a weird issue where these styles for the header elements were overriding the styles defined in the demo app. I can take another look but I couldn't figure out why this was taking precedence. This normalize file is very common and is also used by bootstrap

Reply:
I have had a few issues with the current reset as well, so I am ok with this change. I would like to point out this is just more risky. Lets make sure we validate styles are the same when this gets checked in